scripts/esm/camera-frame.mjs

```diff
@@ -217,6 +219,23 @@ class Grading {
     tint = new Color(1, 1, 1, 1);
 }
 
+/** @interface */
+class ColorLUT {
+    /**
+     * @attribute
+     * @type {Asset}
+     * @resource texture
+     */
+    texture = null;
+
+    /**
+     * @range [0, 1]
```

Comment:
You can use `@visibleif {texture}` which uses truthy values. The parser isn't actually working with null at the moment

Reply:
I tried that as a first thing, but that didn't work. Does it work on your side?